openedx/frontend-app-course-authoring
# feat: add colored left border to items in course outline

**Description:**

Adds colors to left border of each item in course outline based on its publish status. Also, updates the way item status is calculated.

**Screenshot**

![image](https://github.com/openedx/frontend-app-course-authoring/assets/10894099/386e0add-5ad6-4c53-8ef3-8f7ed7d359cd)

**Test instructions:**

* Get devstack up and running using `make {lms,cms}-up`.
* Start npm devserver in this repo using `npm start`.
* Goto http://localhost:2001/course/course-v1:edX+DemoX+Demo_Course/ and see the colors in action.
